tests/fixtures/ingest_helper.py

```diff
@@ -18,6 +19,18 @@ def ingest_file(self, path: Path) -> IngestResponse:
         ingest_result = IngestResponse.model_validate(response.json())
         return ingest_result
 
+    def ingest_file_with_metadata(self, path: Path, metadata: dict) -> IngestResponse:
+        files = {
+            "file": (path.name, path.open("rb")),
+            "metadata": (None, json.dumps(metadata)),
+        }
```

Comment:
Same as below. Ideally, it would say it has to be something like this:
```
files = {
  "file": (path.name, path.open("rb")),
   "metadata": metadata
}